modal_proto/api.proto

```diff
@@ -2072,6 +2073,11 @@ message TaskStats {
   double started_at = 4;
 }
 
+message TaskTemplate {
+  uint32 priority = 1;
+  Resources resources = 2;
+  uint32 concurrent_inputs = 3;
```

Comment:
how is this going to eventually align with the new target/max concurrency parameterization?

Reply:
https://github.com/modal-labs/modal-client/pull/2184 (`concurrent_inputs` is supposed to be the `target_` variant).